examples/scigrid-de/add_load_gen_trafos_to_scigrid.py

```diff
@@ -92,7 +92,7 @@
 # note that some columns have 'quotes because of fields containing commas'
 vertices = pd.read_csv(
     folder_prefix + "scigrid-151109/vertices_de_power_151109.csvdata",
-    sep=",",
+    sep=", ",
```

Comment:
This is a separator and csv reading does not like the the space after the comma, could you double-check whether this works as expected?

Reply:
In the version of Pandas I have, whitespace is cleaned up as you would expect. `sep=','` is sufficient.